.env.demo

```
@@ -128,6 +128,21 @@ AWS_S3_STOREOBJECT_BUCKET=
 # Provide bucket used for storing shortened url objects e.g. 'https://s3.region-code.amazonaws.com/bucket-name'
 SHORTENED_URL_DOMAIN='https://s3.AWS_S3_STOREOBJECT_REGION.amazonaws.com/AWS_S3_STOREOBJECT_BUCKET'
 
+# Set to "true" to store files locally in the uploadedFiles directory (uses IS_LOCAL_FS=true)
+# When IS_LOCAL_FS=true, AWS/RustFS credentials are not required
+IS_LOCAL_FS=false
+
+
+RUSTFS_ENDPOINT=http://10.0.0.190:9000
+RUSTFS_ACCESS_KEY_ID=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUSTFS_ACCESS_KEY_ID key should go before the RUSTFS_ENDPOINT key

(UnorderedKey)